src/interfaces/stepper-interfaces/AddPeopleAndRole/people-and-roles-resource-interface.ts

```diff
@@ -0,0 +1,19 @@
+import { Rules } from '@/enums'
+
+interface Rule {
+  rule: Rules
+  text: string
+  test: (...args) => boolean // rule function
```

Comment:
NB: `test` is a function, with a variable number of arguments, that returns a boolean.